### DISCUSSION TITLE
Issue #4: Removing References to parent repo

### DISCUSSION THREAD

**Discussion by kaushikjadhav01** [NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11#57] [General]
@Surya-06 Please check and close issue https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/issues/4 . Open since quite sometime.

**Comment by surya-prakash-susarla** [comment]
I think I've done this a week ago. Just updating here since it's open.
